src/lib.rs

```diff
@@ -434,7 +457,14 @@ pub trait PolynomialCommitment<G: AffineCurve>: Sized {
         fs_rng: &mut Self::RandomOracle,
         labeled_randomnesses: impl IntoIterator<Item = &'a LabeledRandomness<G, Self::Randomness>>,
         mut rng: Option<&mut dyn RngCore>,
-    ) -> Result<(Polynomial<G::ScalarField>, G::ScalarField, Self::Randomness, bool, Self::Commitment), Self::Error>
+    ) -> Result<(
+            Polynomial<G::ScalarField>, // the polynomial h(X)
+            G::ScalarField,  // the new random challenge challenge 
+            Self::Randomness, // the (optional) commitment randomness of h(X)
+            bool, // `is_hiding` for the batch. Is `true` if one of the involved polynomials needs hiding.
+            Self::Commitment // the commitment of h(X)
+        ), 
+        Self::Error>
         where
             Self::Randomness: 'a,
             Self::Commitment: 'a
```

Comment:
It is not completely clear to me the role  of poly_map, rands_map, commitment_map in the function. Can you do some comment about them?